### ISSUE TITLE
Error Saving Game

### ISSUE THREAD

**Issue by samexists** [Northfear/fallout2-ce-vita#22]
I get this error message when I try to save my game on the vita. “Error Saving Game! Unable to Save Game.” I tried deleting and recreating the SAVEGAME folder but that didn’t fix it.

**Comment by Northfear** [comment]
Going to need a bit more info on this one. It it reproducing in 100% of cases?
Was it failing to save right from the start? Or did something happened in game?
Are you using a premade character or a custom one? Any weird plugins installed?

**Comment by samexists** [comment]
At first I had created a custom character, and I was at the first area you see in the game. After it failed to save, I tried one of the pre made characters and it still failed to save.  It’s reproducible on my end at least, I haven’t seen this bug reported anywhere I searched. 

I have these plugins installed

Osyscall6 by SKGleba

NoNpDrm by TheOfficialFlow

NoNpDrm Unofficial by LMAN

No Power Limits by Electry

pngshot by xyzz

PSVshell by Electry

Repatch by dots-tb 

Shellbat by nowrep 


The version I downloaded was from VitaDB

https://vitadb.rinnegatamante.it/#/

**Comment by Okamiden15** [comment]
I'm having g this same issue on both fallout 1 and fallout 2.

**Comment by Northfear** [comment]
Latest development build? There were some fixes related to failed saves recently

**Comment by Okamiden15** [comment]
Yeah, dev build on both games, unable to save on both. On fallout 2 I can move between areas, but just can't save

**Comment by Northfear** [comment]
Can I get a list of installed plugins too?

**Comment by Okamiden15** [comment]
Sure, this is the order in my Tai folder:
Custom boot splash
Custom warning
Download Enabler
Ds3 vita
Ds4 vita
Henkaku
Kubridge
Kuio
Lolita500
Nonpdrm
Nopsmdrm
Pngshot
ReFOOD
Repatch
Repatch_4
Shellbat
Taihen
Turbopad
Vitacheat360
Yamt

**Comment by Northfear** [comment]
And fallout data folder is located on ux0 partition? Or some other one?

**Comment by Okamiden15** [comment]
Yup, uxo/app/FOUT00001/ for fallout 1, and uxo/app/FOUT00002 for fallout 2

**Comment by Northfear** [comment]
No, I mean is it `ux0:data/fallout2`? Or is some kinda of re-linking to external card partition is used? (uma0, imc0)
I think rePatch can be used for that. Could you try disabling Repatch?

**Comment by Okamiden15** [comment]
I have an sd2vita installed that is used as my main storage. So it's labeller as my uxo, and all of my game sget installed there. When I installed the vpk for the games, they installed in the app folder

**Comment by Okamiden15** [comment]
And I disabled Repatch, anD tried again, still can't save in either game, and can't leave the vault 13 cave in 1

**Comment by Northfear** [comment]
Ok, one more question. What game version (data files) are you using? And what language?

**Comment by Okamiden15** [comment]
I'm using the US GOG version, English

**Comment by Okamiden15** [comment]
Ask whatever questions you need, if it'll help others whoay experience the same issues I'll give whatever help you need man

**Comment by Northfear** [comment]
Ok, let's start with a game data. Could you upload your `ux0:data/fallout/` (or fallout2) folder somewhere?
I want to make sure that the game data is the same before trying out a bunch of plugins.

**Comment by Okamiden15** [comment]
Sure, where the easiest place to upload it for you?

**Comment by Okamiden15** [comment]
OK so I'm just really stupid. This is the first game I've played on vita that requires additional game files, so I didn't know I had to make a whole new folder in the data folder, and I put the game files in the actual installation folder in the apps folder. I made the fallout and fallout 2 folders in uxo/data, and that solved my problem. Sorry! Hopefully that helps the original post author too, if they hadn't already fixed it.
